msteams-platform/get-started/deploy-csharp-app-studio.md

```diff
@@ -19,33 +19,33 @@ Complete the following steps to update the app package:
 
 1. To install App Studio in Teams, select the **Apps** icon at the bottom of the left-hand bar, and search for **App Studio**:
 
-    <img  width="450px" alt="Finding App Studio in the Store View" src="~/assets/images/get-started/searchforAppStudio.png"/>
+    :::image type="content" source="~/assets/images/get-started/searchforAppStudio.png" alt-text="Finding app studio in the store view.":::
```

Comment:
Again, what casing should `app studio` word have?

Reply:
changed the case